ThunderCloud/StormLanguageController.swift

```diff
@@ -95,19 +155,24 @@ public class StormLanguageController: NSObject {
     private func languagePacks() -> (regionalLanguagePack: LanguagePack?, majorLanguagePack: LanguagePack?)? {
         
         //Find out if any locales match
-        guard let _availableLocales = availableLocales, let _preferredLocales = preferredLocales else {
+        guard let _availableLanguagePacks = availableLanguagePacks, let _preferredLocales = preferredLocales else {
```

Comment:
I've been trying to move away from this `_` syntax when not needed, in this case there's no reason why they're needed :) Sometimes they are, and when they are we should use the underscore :D

Reply:
I agree 100%, I almost always shadow the variable but I'll be honest I did a find and replace for this 😅

Reply:
Hahaaa 😝 makes sense now! Yeah when I re-wrote Storm in swift I changed all of the `_` to just shadow the variable 👍